tests/lightning_qubit/test_state_vector_class.py

```diff
@@ -24,21 +24,29 @@
 from pennylane.tape import QuantumScript
 from pennylane.wires import Wires
 
-from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
+if device_name == "lightning.qubit":
+    from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
 
 if device_name == "lightning.kokkos":
-    pytest.skip("Kokkos new API in WIP.  Skipping.", allow_module_level=True)
+    from pennylane_lightning.lightning_kokkos._state_vector import (
+        LightningKokkosStateVector as LightningStateVector,
+    )
```

Comment:
Should we put that sort of logic in `conftest` so we can just use a generic name like `StateVector`?

Reply:
I think we can do that but I suggest keeping this like that to move forward with this PR and changing it before merging with `master`